libcontainer/container_linux.go

```diff
@@ -2109,6 +2165,22 @@ func (c *linuxContainer) bootstrapData(cloneFlags uintptr, nsMaps map[configs.Na
 		Value: c.config.RootlessEUID,
 	})
 
+	// Bind mount source to open.
+	if it == initStandard && c.shouldSendMountSources() {
+		var mounts []byte
+		for _, m := range c.config.Mounts {
+			if m.IsBind() {
+				mounts = append(mounts, []byte(m.Source)...)
+			}
+			mounts = append(mounts, byte(0))
```

Comment:
The `byte()` conversion is not needed, as `0` is an untyped numeric constant that can be used as `byte` as is.

Reply:
heh, will leave it like it is, then :D